bismuthcore/bismuthcore/legacy/bismuthwallet.py

```diff
@@ -44,10 +44,11 @@ def wallet_preview(self, wallet_file='wallet.der'):
             try:
                 key = RSA.importKey(content['Private Key'])
                 info['encrypted'] = False
-            except:  # encrypted
+            except Exception:  # encrypted
+                print("Wallet is encrypted")
                 info['encrypted'] = True
-        except:
-            pass
+        except Exception as e:
+            print("Unknown error occurred during wallet preview. Error: {}".format(e))
```

Comment:
What is the rationale for prints instead of the self.app_log used everywhere?

Reply:
`self.app_log` is not an available attribute. Also, I intended to ignore `legacy/`, but failed to ignore it when I did a search for `except:` across the codebase. I may look into removing these commits in `legacy/` in order to avoid confusion.